### ISSUE TITLE
Request - ctlithostratigraphic_formation (or CTs in general?) rules/formatting

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#9846]
**Describe what you're trying to do** 

Understand a change, probably update authority documentation.

----------

I think https://github.com/ArctosDB/arctos/issues/9836 is suggesting something for https://handbook.arctosdb.org/documentation/authorities.html but I can't quite understand what.

The original issue (see below) says "There is also a Prince Albert Formation in Canada according to macrostrat." - I suspect this is what lead to the disambiguator, but that doesn't exist in Arctos, Google doesn't readily find it, and I suspect there are lots of other homonyms out there.(Lacking some registry, there always are!)

Should the documentation involve "add an ``of {place}`` clarifying term if ambiguity is known to exist", or should that be limited to when the term is in Arctos, or ??? Terms are required to be unique but we have a definition and the term itself doesn't need to carry that. ??? How can we standardize this? (I think I'd probably drop the "of South Africa" in this case, but I do think some things need this, we could use a recipe, and all I really care about is standardization/consistency, which should help with avoiding messes.)

--------------

Semi-related:

Should this sort of thing have a roadmap? If not, can anything else skip that step?

Also I had to dig through the changelog to find the original issue (https://github.com/ArctosDB/arctos/issues/7974), which was removed for the update, I suspect that was just an oopsie and it should be added back?

@WaigePilson 

------------

Please review the [Arctos Code of Conduct](https://arctosdb.org/code-of-conduct/) to ensure respectful communication, and see <https://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html> for more information about submitting and managing Issues.

**Comment by WaigePilson** [comment]
@dustymc the paleo sub committee has discussed this at length, and decided that for formations in countries outside of North America we will add the country if there is another formation in North America with the same/very similar name. I think there are only really two examples of this right now (Blackstone Formation in Australia, Prince Albert Formation in South Africa)--both of those names just missed the "of" in their code table value, which we (the paleo sub committee) decided was our preference for this code table. Both of those issues (linked below) are completed, no necessary steps for you!

https://github.com/ArctosDB/arctos/issues/9844
https://github.com/ArctosDB/arctos/issues/9836

So those issues are closed, not sure what else you want from this issue?

**Comment by ekrimmel** [comment]
Perhaps the ask here is for @WaigePilson (on behalf of or in addition to other members from @ArctosDB/paleontology) to [review the "Specific Rules for Specific Code Tables" documentation that Dusty maintains on the Handbook Authorities page](https://handbook.arctosdb.org/documentation/authorities.html#specific-rules-for-specific-code-tables)? Text was moved from individual lithostrat code tables for consolidation/more consistent documentation reasons, and the version of the text on the individual code tables had been through Paleo Committee review, but more work in the committee has happened since then and this may be a good opportunity to make sure the Committee's conventions are recorded in Arctos-wide documentation. E.g. adding something like Dusty's suggested "add an of {place} clarifying term if ambiguity is known to exist" to record the Committee's consensus around values like "Prince Albert Formation of South Africa."
